src/json.rs

```diff
@@ -12,7 +16,7 @@ pub enum JsonValue {
     Object(BTreeMap<String, JsonValue>),
 }
 
-#[derive(Debug)]
+#[cfg_attr(test, derive(PartialEq))]
```

Comment:
I would add more than just PartialEq. Also, does it only need to be for test? If it isn't used, then it'll not even be compiled so it is fine to add more than enough here than just what is needed. Typically for enums I would add like (off the top of my head here) `Debug, Ord, PartialOrd, Eq, PartialEq` which should cover all that we would use.

So, drop `cfg_attr`, add more derives

Reply:
Redundant derives IMHO is a bad style. Also for the release version, it's completely wrong.

Reply:
I don't necessarily agree as Rust compiler is smart enough to know which to compile. It won't add to the byte size of the resulting WASM.

Reply:
@sept-en that was from you. What do you think?